app/src/main/java/com/li/weatherapp/utils/AirPollutionUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.li.weatherapp.utils
+
+import android.content.Context
+import com.li.weatherapp.R
+
+object AirPollutionUtils {
+    
+    fun getAirPollutionTitle(aqiDegree: Int, context: Context) = when (aqiDegree) {
+        in 0..19 -> context.resources.getString(R.string.title_wonderful)
+        in 20..49 -> context.resources.getString(R.string.title_good)
+        in 50..99 -> context.resources.getString(R.string.title_bad)
+        in 100..149 -> context.resources.getString(R.string.title_harmful)
+        in 150..249 -> context.resources.getString(R.string.title_very_harmful)
+        else -> context.resources.getString(R.string.title_dangerous)
+    }
+
+    fun getAirPollutionDescription(aqiDegree: Int, context: Context) = when (aqiDegree) {
```

Comment:
sai conventions 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#29-parameter-ordering-in-methods